en/edition-1.md

```diff
@@ -0,0 +1,194 @@
+# First edition of NomadConf.eu
+
+The first edition took place on Thursday 17 August 2017 and Friday 18 August, between Paris and Bordeaux. Participants were [@matti_sg](https://twitter.com/matti_sg), [@pblayo](https://twitter.com/pblayo), [@krichtof](https://twitter.com/krichtof) and [@cranedemorse](https://twitter.com/cranedemorse).
+
+## What happened
+
+### Departure from Paris
+
+Departure by train at 8:30 AM. We meet right in the car. Krichtof misses the train after an alarm failure and takes the 8:53 AM train.
+
+### Paris → Bordeaux
+
+Informal breakfast at the TGV bar to meet and define the schedule for the 3-hours trip.
+
+First exercise:
+
+1. Tell us your expectations for these two days.
+
+> To learn from this experience, and figure out how to work remotely in the future and charge for the time worked on the train.
+>
+> Meet new people.
+
+2. Write at least two a priori that you have on working in the train (by completing the sentence “I think that working in the train…”). Don't share them!
+
+![TGV Paris Bordeaux](/img/edition-1/IMG_5421.jpg)
+
+We start working, each one in his or her place, at 8:55am. We set up a meeting at 9:55 in the bar to share:
+
+- The biggest obstacle we encountered (which prevented us from being as productive as when we are sedentary).
+- A surprise we had.
+
+We try making a conference call _between two high-speed trains_ by involving @Krichtof from his own train. The result is not convincing, we move on to Slack's written channel.
+
+#### Impediments
+
+> Not being able to access the internet, for doc search. I managed to get around it by reading the code I already had on my laptop.
+>
+> Nothing to do with the mobility situation: I have several sessions on my computer and switching between my personal and professional sessions I missed some important interactions.
+>
+> Nothing to do with the mobility situation either: the project I'm working on comes to life around 10am, so this first iteration was in a way “disconnected” from the teams. I didn't get enough sleep last night, so it was very hard to read (the activity I switched to for this first iteration).
```

Comment:
I guess it's OK to write 10am rather than 10AM? Below the case is 4:34PM, but I'm the kind of person who's startled when I read 'i' rather than 'I' from a native speaker.